cypress/integration/Button.spec.ts

```diff
@@ -7,7 +7,7 @@ describe('Button', () => {
 
   context('given primary buttons are rendered', () => {
     beforeEach(() => {
-      h.stories.load('button', 'primary');
+      h.stories.load('components-buttons-button-react', 'primary');
```

Comment:
These should have been capitalized before. The intent was to match the exact string located in the `storiesOf` and `add` functions to make it easier without having to translate manually the URL.

I'll adjust and see what we think.

Reply:
Now this line reads:
```ts
h.stories.load('Components|Buttons/Button/React', 'Primary');
```